content/02.introduction.md

```diff
@@ -1,29 +1,45 @@
+In spite of the success and excitement of preprints, there are number of issues that arose from their constant use.
+Preprint repositories receive an enormous amount of preprint submissions which results in a number of functionality issues [@doi:10.1371/journal.pgen.1008565].
+For example, repositories have a hard time searching and linking preprints with their published counterparts [@doi:10.1038/530265a; @doi:10.7554/eLife.52646], which results in a missing links within the repository.
+Furthermore, these repositories lack tools to show how textual content of preprints are altered due to the peer review process [@doi:10.1371/journal.pgen.1008565].
+These repositories are open access for all to view preprints, which results in concern from scientists that they could be scooped by competitors [@doi:10.1371/journal.pgen.1008565; @doi:10.1371/journal.pbio.3000151].
+Plus, preprint repositories do not have in-depth peer review which can result in posted preprints containing inconsistent results or conclusions [@doi:10.12688/f1000research.19619.2; @doi:10.1007/s10393-018-1352-3; @doi:10.1038/530265a; @doi:10.1016/j.bpj.2016.06.035].
+Despite these discoveries, one piece information that has yet to be examined is the textual content contained within the bioRxiv repository.
 
-We sought to understand the language landscape of preprints by performing a linguistic analysis of the bioRxiv corpus.
+Textual analysis is a methodology that uses linguistic, statistical and machine learning techniques to analyze and extract information from text [@doi:10.1111/1475-679X.12123]. 
+This set of techniques have made a sizable impact within the life science community by providing valuable insight on biomedical text.
+For instance, scientists analyzed linguistic similarities and differences of biomedical corpora [@doi:10.1186/1471-2105-10-183;10.1186/1471-2105-9-S3-S6; @pmc:PMC442180].
+Scientists have provided the community with a number of tools that aide future text mining systems [@doi:10.1093/bib/bbs084; @doi:10.1093/nar/gkz389; @doi:10.1186/s12859-017-1775-9] as well as advice on how to train and test future text processing systems [@doi:10.1186/1471-2105-11-492; @doi:10.1186/1471-2105-13-207; @doi:10.1186/s12859-019-2604-0].
+Due to the previous successes within this field, we sought to use this type of methodology for analyzing the bioRxiv repository.
+Plus, to our knowledge this is the first time textual analysis has been conducted on the bioRxiv repository.
+
+In this work we sought to analyze the textual content contained within the bioRxiv repository.
+Through this work we can contextualize how this language changes under the guise of peer review and provide initial insight on which sections are altered.
+Furthermore, this work provides an opportunity to improve repository functionality and motivate future ideas on augmenting preprint repository capabilities.
+
+We accomplished the above goals through a set of analyses.

```

Comment:
This paragraph is more or less a list of what you did. Is there a way to get a slightly zoomed out paragraph (not so much on the specific analyses) that talks very briefly about why + hints at what you learned and that uses what you did in support of that? I think of this paragraph a bit like the trailer for the paper.